app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.ProductService;
+import com.codesoom.assignment.domain.Product;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * 상품에 대한 요청을 받고 응답을 보낸다.
+ */
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/products")
+@CrossOrigin
+public class ProductController {
+
+    private final ProductService productService;
+
+    /**
+     * 상품목록 조회 요청은 상품목록을 리턴한다.
+     * @return 상품목록
+     */
+    @GetMapping
+    public List<Product> getProducts() {
+        return productService.getProducts();
+    }
```

Comment:
이 메소드는 "상품목록 조회 요청"이 아니에요. "상품목록 조회 요청"이 들어왔을 때 그 요청을 처리해주는 핸들러죠.
```suggestion
    /**
     * 상품목록을 리턴한다.
     * @return 상품목록
     */
    @GetMapping
    public List<Product> getProducts() {
        return productService.getProducts();
    }
```

Reply:
네 맞아요. 주석을 작성할 때 순간 요청이라는 개념에 집중했나 보네요